ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -567,3 +567,36 @@ def get_blockpools_compression_status_from_storagesystem(self) -> tuple:
             self.validation_loc["storagecluster-blockpool-details-compress-status"]
         )
         return compression_status_blockpools_tab, compression_status_blockpools_details
+
+    def verify_odf_operator_in_installed_operator(self) -> bool:
+        """
+        This function checks that post ODF installation only ODF operator is present and not also
+        OCS operator in the Installed Opertor tab.
+        This function is only written for 4.9+ versions
+
```

Comment:
Its confusing does this mean. Function validates post ODF installation only ODF operator should be present in Operator Tab and OCS operator shouldn't be there?

Reply:
Yes it does that